admin-manual/maintenance/cli-import-export.rst

```diff
@@ -1642,6 +1643,224 @@ are intentionally importing duplicate deaccession records, you can use the
 
 :ref:`Back to top <cli-import-export>`
 
+.. _csv-import-storage-cli:
+
+Import physical storage containers and locations
+------------------------------------------------
+
+This task will allow for the import of physical storage data, as well as updates 
+to existing physical storage containers, via an accompanying CSV file. Read more 
```

Comment:
this is perhaps overly pedantic, but "accompanying CSV file" sounds like the import is accompanied by a CSV file - I think you can simply remove the word "accompanying"